applications/luci-app-xinetd/htdocs/luci-static/resources/view/xinetd/xinetd.js

```diff
@@ -71,26 +97,29 @@ return view.extend({
 		o.depends('type', 'UNLISTED');
 		o.rmempty = false;
 		o.modalonly = true;
-		o.validate = function(section_id, value) {
-			var sections = uci.sections('xinetd', 'service');
-
-			for (var i = 0; i < sections.length; i++) {
-				if (uci.get('xinetd', sections[i]['.name'], 'port') == value && section_id != sections[i]['.name'])
-					return _('Port already in use by service "%s"'.format(sections[i]['.name']));
-			}
-
-		return true;
-		};
+//		//FIXME: does not work, support for tcp/udp missing and also for INTERNAL
+//		o.validate = function(section_id, value) {
+//			var sections = uci.sections('xinetd', 'service');
+//
+//			for (var i = 0; i < sections.length; i++) {
+//				if (uci.get('xinetd', sections[i]['.name'], 'port') == value && section_id != sections[i]['.name'])
+//					return _('Port already in use by service "%s"'.format(sections[i]['.name']));
+//			}
+//
+//		return true;
+//
+//		};
```

Comment:
If this is not working any more, then we should delete the whole validate function or we should fix them.

Reply:
This never worked, even before these commits if there is a tcp port used and the same port as udp (could be different daemons, does not matter).
And now additional it does not work for a service name read from /etc/services. So it is possible to fix it.
"Impossible" is it to know if some other daemon uses it. Maybe during configuration, but not if some additional packages are later installed

On the other side xinetd tell in syslog if a port could not be used - if misconfigured or some other daemon uses it already, and does not start this single service

Reply:
Well then we delete the whole thing